docs/src/route.ts

```diff
@@ -31,6 +31,7 @@ const ROUTES = {
   goldSponsor: '/material-ui/discover-more/backers/#gold',
   store: 'https://mui.com/store/',
   advancedComponents: '/x/introduction/',
+  advancedComponentsWhatsNew: '/x/whats-new',
```

Comment:
Shouldn't it link to `https://next.mui.com/x/whats-new`? There won't be a `mui.com/x/whats-new` page until we release v6 stable.

Reply:
> There won't be a mui.com/x/whats-new page until we release v6 stable.

That was my initial thought, but I'm wondering why not keep it after?

Reply:
We can keep it after, but if we deploy this banner today, it will point to `mui.com/x` which currently hosts v5, while the "What's new" page will be available in v6 docs (next.mui.com).
This will result in a broken link if I understand it correctly.

Reply:
Got it! Nice catch!

Reply:
Done!